src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import DisplayCalculator from './DisplayCalculator';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="App">
+    <h2>My React Calculator!!!</h2>
+    <DisplayCalculator />
+    <ButtonPanel />
+  </div>
```

Comment:
You need to wrap both `DisplayCalculator` and `ButtonPanel` with [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax)
Reason:
We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `DisplayCalculator` and `ButtonPanel`.

![image](https://user-images.githubusercontent.com/31897434/118618136-391e8180-b7bb-11eb-9a16-4fe755ce9f2c.png)